test/e2e-oidc/external_oidc_test.go

```diff
@@ -548,7 +548,7 @@ func (tc *testClient) updateAuthResource(t *testing.T, ctx context.Context, base
 }
 
 func (tc *testClient) checkPreconditions(t *testing.T, ctx context.Context, authType *configv1.AuthenticationType, caoStatus []configv1.ClusterOperatorStatusCondition, kasoStatus []configv1.ClusterOperatorStatusCondition) {
-	err := wait.PollUntilContextTimeout(ctx, 10*time.Second, 2*time.Minute, false, func(ctx context.Context) (bool, error) {
+	err := wait.PollUntilContextTimeout(ctx, 30*time.Second, 20*time.Minute, false, func(ctx context.Context) (bool, error) {
```

Comment:
Since we're increasing the timeout here, would it make sense to bump the timeout of the whole test job in the same PR as well?

https://github.com/openshift/cluster-authentication-operator/blob/master/Makefile#L84

Reply:
Ah, yes it would - good catch :)